src/components/BotBuilder/BotWizard.js

```diff
@@ -76,6 +75,11 @@ class BotWizard extends React.Component {
       imageChanged: false,
       loaded: false,
       commitMessage: '',
+      skillCategory: null,
+      skillLanguage: '',
+      skillName: '',
+      code:
```

Comment:
Please revert this back to `buildCode` rather than `code` so that we know that which tab does it belong to. I think it make things more clear.